libs/@guardian/source-foundations/src/colour/palette.ts

```diff
@@ -1,24 +1,8 @@
-import { tokens } from '@guardian/design-tokens';
-
 /**
  * [Storybook](https://guardian.github.io/csnx/?path=/docs/source-foundations_palette--palette) •
  * [Design System](https://theguardian.design/2a1e5182b/p/492a30-light-palette)
  *
  * Colour palette organised by type e.g. 'error', or pillar e.g. 'opinion'
  */
-export const palette = {
-	brand: tokens.palette.brand,
-	brandAlt: tokens.palette.brandAlt,
-	neutral: tokens.palette.neutral,
-	error: tokens.palette.error,
-	success: tokens.palette.success,
-	news: tokens.palette.news,
-	opinion: tokens.palette.opinion,
-	sport: tokens.palette.sport,
-	culture: tokens.palette.culture,
-	lifestyle: tokens.palette.lifestyle,
-	labs: tokens.palette.labs,
-	specialReport: tokens.palette.specialReport,
-	specialReportAlt: tokens.palette.specialReportAlt,
-	focus: tokens.palette.focus,
-} as const;
+
+export { palette } from '@guardian/design-tokens';
```

Comment:
❤️

Reply:
I remember why we did this. It was for the ordering in chromatic

Reply:
Ah, yeah, that was probably it. I don't think the ordering in Chromatic matters so long as the colours are correct. This is a lot more concise than constructing a new palette object.